api/v1alpha1/etcd_types.go

```diff
@@ -393,3 +393,92 @@ type EtcdList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Etcd `json:"items"`
 }
+
+// LastOperationState is a string alias.
+type LastOperationState string
+
+const (
+	// LastOperationStateActive indicates that an operation is ongoing.
+	LastOperationStateActive LastOperationState = "Active"
```

Comment:
```suggestion
	LastOperationStateProcessing LastOperationState = "Processing"
```
To be consistent with `g/g`.